### PR TITLE
Refactored the script litmus_job_runner to get the name of litmus pod 

### PR DESCRIPTION
**Pull request by sushma1118**
* The script is modified to get the name of litmus pod and if pod is not found, it tries again till retry_count 
  becomes zero.
* The following script is modified:
      utils/litmus_job_runner

The bash script is tested in Openshift cluster manually by editing litmus job name.
* If the litmus job name is correct, then pod name is found and  the program exits from while loop
```
[root@master-1554817485 deployers]# ./test1.sh

```
* If the job name is wrong, then pod name is not found and the script tries again to get the name till 
  retry_count becomes 0. Finally exits from while loop by printing appropriate error message.
```
[root@master-1554817485 deployers]# ./test.sh
unable to get litmus pod name....trying again
unable to get litmus pod name....trying again
unable to get litmus pod name....trying again
unable to get litmus pod name....trying again
unable to get litmus pod name....trying again
unable to get litmus pod name....trying again
unable to get litmus pod name....trying again
unable to get litmus pod name exiting

```

### CODE REVIEW

File: Openshift-EE/utils/litmus_job_runner [hunk 1]
```diff
@@ -22,9 +22,27 @@ bash ../Openshift-EE/utils/error_handler ${retcode} msg="Unable to run litmusboo
...
+  if [[ $rc -eq 0 ]]; then
+    if [[ -z ${litmus_pod} ]]; then   # if pod is not found it returns null. If pod name is not got sleep for 10 sec and try again.
+      sleep 10
+      retry_count=$((retry_count-1))
+      echo "unable to find test runner pod ....trying again"
+      if [[ $retry_count -eq 0 ]]; then
+        echo "unable to find test runner pod exiting"  # retry count is over.. pod is not found.. exit from while loop
+        exit 1;
+      fi
+    else
```
Comment (gprasath) [inline Openshift-EE/utils/litmus_job_runner]: remove else block

File: Openshift-EE/utils/litmus_job_runner [hunk 2]
```diff
@@ -22,9 +22,27 @@ bash ../Openshift-EE/utils/error_handler ${retcode} msg="Unable to run litmusboo
...
-litmus_pod=$(eval ${litmusPodCmd}); retcode=$?
-bash ../Openshift-EE/utils/error_handler ${retcode} msg="Unable to find litmus test runner pod, exiting" action="exit"
 
+# Loop till you get the running litmus pod. If not able to find the pod, print error message
+let retry_count=7   # number of times you want to try again to get the name of litmus pod
+while true; do
+  litmus_pod=$(eval ${litmusPodCmd}); rc=$?
+  if [[ $rc -eq 0 ]]; then
+    if [[ -z ${litmus_pod} ]]; then   # if pod is not found it returns null. If pod name is not got sleep for 10 sec and try again.
+      sleep 10
```
Comment (gprasath) [inline Openshift-EE/utils/litmus_job_runner]: reduce the interval to 5s

File: Openshift-EE/utils/litmus_job_runner (hunk 1 above)
Comment (sushma1118) [reply Openshift-EE/utils/litmus_job_runner]: Done. Removed the else block

File: Openshift-EE/utils/litmus_job_runner (hunk 2 above)
Comment (sushma1118) [reply Openshift-EE/utils/litmus_job_runner]: Reduced to 5s

**Pull request closed without merging**